src/LibraryManager/Helpers/JsonHelpers.cs

```diff
@@ -1,15 +1,69 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.Web.LibraryManager.Contracts;
-using Microsoft.JSON.Core.Parser.TreeItems;
-using System.Linq;
 using System.Collections.Generic;
+using System.Linq;
+using Microsoft.JSON.Core.Parser;
+using Microsoft.JSON.Core.Parser.TreeItems;
+using Microsoft.Web.LibraryManager.Contracts;
 
-namespace Microsoft.Web.LibraryManager.Vsix
+namespace Microsoft.Web.LibraryManager.Helpers
 {
     internal static class JsonHelpers
     {
+        internal static JSONParseItem GetItemBeforePosition(int pos, JSONComplexItem parentItem)
```

Comment:
>JSONParseItem [](start = 24, length = 13)

The moved files should remain in the VSIX project. 
For adding the tests, you can add new Test project which refers the VSIX project.

Reply:
Yes. will do

---
In reply to: [203547823](https://github.com/aspnet/LibraryManager/pull/324#discussion_r203547823) [](ancestors = 203547823)